.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
+local.properties
```

Comment:
`.idea` folder should be completely ignored
You can take [.gitignore](https://github.com/ARK-Builders/ARK-Navigator/blob/main/.gitignore) from the Navigator app
Also these files should be removed from this branch